frontend/packages/noobaa-storage-plugin/integration-tests/views/obcPage.view.ts

```diff
@@ -73,11 +112,60 @@ export class CreateOBCHandler {
     return { name, status };
   }
 
+  async waitUntilBound() {
```

Comment:
Can we use this the way we are using the `isDeploymentReady` function, it would be nice to have all the waiting mechanism coherent in our tests?